markets/perps-market/contracts/storage/AsyncOrder.sol

```diff
@@ -74,12 +143,21 @@ library AsyncOrder {
         self.accountId = commitment.accountId;
     }
 
+    /**
+     * @notice Resets the order.
+     * @dev This function is called after the order is settled or cancelled.
+     * @dev Just setting the sizeDelta to 0 is not enough, since is the value checked to identify an active order at settlement time.
```

Comment:
> Just setting the sizeDelta to 0 is not enough, since is the value checked to identify an active order at settlement time.

Do we need to update the implementation to reset settlement time too?

Reply:
thanks for spotting it! copypasta error... the NOT shouldn't be there